internal/apps/cmp/initialize.go

```diff
@@ -78,7 +78,9 @@ func (p *provider) initialize(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	p.Router.Any("/**", server.Router().ServeHTTP)
+	if err := server.RegisterToNewHttpServerRouter(p.Router); err != nil {
```

Comment:
This may cause kubectl console and pod console 404

Reply:
This used to be good, I will tested it for steve API later.

Reply:
Adjusted, it works now :D